src/stylesheets/common/_lists.scss

```diff
@@ -0,0 +1,39 @@
+article {
```

Comment:
According to @hanbyul-here, due to a conflict with list styles in the navigation (she mentioned this during friendship meeting today). I'm not happy about this either (even if I was part of the decision making last year). I think we can do better than this, but probably outside this PR. (It's just part of the diff here because I've renamed the file.)